log_request_id/middleware.py

```diff
@@ -21,17 +21,25 @@ def process_request(self, request):
         request.id = request_id
 
     def get_log_message(self, request, response):
+        message = 'method=%s path=%s status=%s' % (request.method, request.path, response.status_code)
```

Comment:
I would consider building the template first, and interpolate the values at the end.
Maybe also consider naming your placeholders.